test/LabsTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+namespace Clearbooks\Labs;
+
+use Doctrine\DBAL\Connection;
+
+abstract class LabsTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var Connection
+     */
+    protected $connection;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->connection = Bootstrap::getInstance()->getDIContainer()
+                                     ->get( 'Doctrine\DBAL\Connection' );
```

Comment:
Ah yeah, that would look nicer. I'll change these everywhere, I'm not sure why I did it like this :D